students/becky_larson/lesson02/Fibonacci_Series.py

```diff
@@ -1,88 +1,95 @@
 #!/usr/bin/env python
-#fib(n) = fib(n-2) + fib(n-1)
+"""fib(n) = fib(n-2) + fib(n-1)#"""
+
 
 def fibonacci(n):
-  """Calculates fibonacci series."""
-
-  if(n<0):
-    print("incorrect entry in fibonacci")
-  elif(n==0): 
-    return 0
-  elif(n==1): 
-    return 1
-  else:
-    return fibonacci(n-1)+fibonacci(n-2)
-    
+    """Calculates fibonacci series."""
+
+    if(n < 0):
+        print("incorrect entry in fibonacci")
+    elif(n == 0):
```

Comment:
the parens are redundant  here, and you would typically have a space between if/elif and a paren unless it's a function call.